src/pages/docs/spaces/locations.mdx

```diff
@@ -73,67 +69,78 @@ The following is an example payload of a location event. Information about locat
   }
 }
 ```
+</Code>
 
 The following are the properties of a location event payload:
 
-|_. Property |_. Description |_.  Type |
-| member.clientId | The "client identifier":/docs/auth/identified-clients for the member. | String |
-| member.connectionId | The unique identifier of the member's "connection":/docs/connect. | String |
-| member.isConnected | Whether the member is connected to Ably or not. | Boolean |
-| member.lastEvent.name | The most recent event emitted by the member. Will be one of @enter@, @update@, @present@ or @leave@. | String |
-| member.lastEvent.timestamp | The timestamp of the most recently emitted event. | Number |
-| member.profileData | The optional "profile data":/docs/spaces/avatar#profile-data associated with the member. | Object |
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| member.clientId| The [client identifier](/docs/auth/identified-clients) for the member. | String |
+| member.connectionId| The unique identifier of the member's [connection](/docs/connect). | String |
+| member.isConnected | Whether the member is connected to Ably or not.| Boolean |
+| member.lastEvent.name| The most recent event emitted by the member. Will be one of `enter`, `update`, `present`, or `leave`. | String |
+| member.lastEvent.timestamp | The timestamp of the most recently emitted event.| Number |
+| member.profileData | The optional [profile data](/docs/spaces/avatar#profile-data) associated with the member. | Object |
 | previousLocation | The previous location of the member. | Object |
-| currentLocation | The new location of the member. | Object |
+| currentLocation| The new location of the member.| Object |
 
 <aside data-type='further-reading'>
-<p>Member location subscription listeners only trigger on events related to members' locations. Each event only contains the payload of the member that triggered it. Alternatively, "space state":/docs/spaces/space can be subscribed to which returns an array of all members with their latest state every time any event is triggered.</p>
+Member location subscription listeners only trigger on events related to members' locations. Each event only contains the payload of the member that triggered it. Alternatively, [space state](/docs/spaces/space) can be subscribed to which returns an array of all members with their latest state every time any event is triggered.
 </aside>
 
-h3(#unsubscribe). Unsubscribe from location events
+### Unsubscribe from location events <a id="unsubscribe"/>
 
 Unsubscribe from location events to remove previously registered listeners.
 
-The following is an example of removing a listener for location update events:
```

Comment:
This seems to have been dropped!
"The following is an example of removing a listener for location update events:"